.github/workflows/test.yaml

```diff
@@ -0,0 +1,18 @@
+# docker continuous integration
+# run tests
+---
+name: Test
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container: python:3.7
```

Comment:
Pulled from isacc-messaging, will change

Reply:
```suggestion
    container: python:3.11
```

Since we generally use debian, I try to track the version of python in debian stable. That's 3.11 at the moment, but you may need to use the version from the last debian release (3.9) if something breaks